compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -1794,14 +1794,13 @@ class Typer extends Namer
 
   /** Translate tuples of all arities */
   def typedTuple(tree: untpd.Tuple, pt: Type)(implicit ctx: Context): Tree = {
-    val elems = tree.trees
-    val arity = elems.length
+    val arity = tree.trees.length
```

Comment:
Nitpick: I'd rename `elems1` to `elems` here.

Reply:
Will do